src/SFA.DAS.EmployerIncentives.Domain/IncentiveApplications/IncentiveApplication.cs

```diff
@@ -39,11 +44,18 @@ public void AddApprenticeship(Apprenticeship apprenticeship)
             Model.ApprenticeshipModels.Add(apprenticeship.GetModel());
         }
 
+        public void RemoveApprenticeship(Apprenticeship apprenticeship)
```

Comment:
Can we not just set the required apprenticeships in the application rather than have add and remove methods?  As an analogy if the application had a name you wouldn't have a RemoveName and AddName method.  You'd just set the name so it would be more simple to just set the apprenticeships